target/linux/realtek/files-6.12/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -2355,7 +2355,8 @@ static int rtl9300_sds_10g_idle(int sds_num);
 static void rtsds_930x_patch_serdes(int sds, phy_interface_t mode);
 
 #define RTL930X_MAC_FORCE_MODE_CTRL		(0xCA1C)
-int rtl9300_serdes_setup(int port, int sds_num, phy_interface_t phy_mode)
+int rtl9300_serdes_setup(int port, int sds_num, phy_interface_t phy_mode,
+						 bool swap_rx_polarity, bool swap_tx_polarity)
```

Comment:
Still weird indentation style. Please align it with the last open parenthesis.

(this source code has 1 tab representing 8 spaces)